File: physics/force_equation/dart/force_equation.dart

/// Function to calculate the force given the mass and acceleration of an object.
double forceEquation(double mass, double acceleration) {
  double force = mass * acceleration;
  return force;
}

main() {
  // Demo:
  double force = forceEquation(20.0, 15.0);
  print('force for an object with mass of 20.0, and an acceleration of 15.0 = $force');
}

File: physics/force_equation/CPP/force_equation.cpp
#include<iostream>
#include<math.h>

using namespace std;

int main(){
	double mass, acceleration;
	cout<<"Enter Mass : ";
	cin>>mass;
	cout<<endl<<"Enter Acceleration : ";
	cin>>acceleration;
	cout<<"Force applied is  :"<<mass*acceleration<<"N";
	return 0;
}
